packages/legend-extension-dsl-mastery/src/graph-manager/protocol/pure/v1/model/packageableElements/mastery/V1_DSL_Mastery_Runtime.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Hashable, hashArray } from '@finos/legend-shared';
+import { MASTERY_HASH_STRUCTURE } from '../../../../../../../graph/DSL_Mastery_HashUtils';
+import { V1_PackageableElement } from '@finos/legend-graph';
+
+export abstract class V1_Runtime
+  extends V1_PackageableElement
```

Comment:
Isn't it called `MasteryRuntime`? If that's the case then can you please rename it to `MasteryRuntime` everywhere

Reply:
The convention that we've used in the rest of the protocols is the same as this one. So, stuck to V1_Runtime. Shall I switch to MasteryRuntime?

Reply:
For protocol models can you modify it to `V1_MasteryRuntime` and for metamodels `MasteryRuntime`